allensdk/brain_observatory/behavior/data_files/sync_file.py

```diff
@@ -43,14 +43,19 @@ class SyncFile(DataFile):
     streams collected during a behavior + ophys session.
     """
 
-    def __init__(self, filepath: Union[str, Path]):
-        super().__init__(filepath=filepath)
+    def __init__(self, filepath: Union[str, Path], permissive: bool = False):
+        self._filepath: str = safe_system_path(str(filepath))
```

Comment:
This is not correct since `super` is not called. Please set `_permissive` then call `super`. `load_data` should pass `self._permissive`

Reply:
How would `load_data` end up passing permissive? `load_data` is a static method and the `super` for the class only passes the file name.